src/internal/m365/collection/exchange/collection_test.go

```diff
@@ -333,6 +336,221 @@ func (suite *CollectionUnitSuite) TestPrefetchCollection_Items() {
 	}
 }
 
+func (suite *CollectionUnitSuite) TestPrefetchCollection_Items_skipFailure() {
+	var (
+		start         = time.Now().Add(-time.Second)
+		statusUpdater = func(*support.ControllerOperationStatus) {}
+	)
+
+	table := []struct {
+		name               string
+		category           path.CategoryType
+		handler            backupHandler
+		added              map[string]time.Time
+		removed            map[string]struct{}
+		expectItemCount    int
+		expectSkippedCount int
+	}{
+		{
+			name:     "no items",
```

Comment:
nit: since other tests are prefixed with the service name could you please prefix this one as well?
```suggestion
			name:     "events no items",
```